src/app/content/highlights/components/EditCard.tsx

```diff
@@ -266,6 +266,8 @@ function AnnotationEditor({
   const initializeColor = React.useCallback(() => {
     if (!props.highlight.getStyle()) {
       onColorChange(highlightStyles[0].label, true);
+      textarea.current?.blur();
+      textarea.current?.focus();
```

Comment:
This on purpose or just for testing?

Reply:
This is on purpose. Focus wasn't sticking without it.

Reply:
Was the focus not sticking because of a re-render? This seems like a render or timing issue we should dig into

Reply:
It seems to be the same issue as [this](https://stackoverflow.com/questions/3685242/unable-to-type-in-textbox-even-with-focus-set): the focus is set, but keyboard input is ignored until you click on the field. Their solution was the same as mine: blur, then focus again (only they used a timeout, which doesn't seem to be necessary).
It only happens when the highlight is a new selection. If I disable the call to `onColorChange`, I am able to type.

Reply:
This issue is the updating of the DOM that `onColorChange` does. In particular, if something is removed from the DOM, focus is lost (although it seems to be only half-lost in this case -- the focus outline remains, it just doesn't direct keyboard input there)